tests/language-feature/extensions/extension-method-simple.slang

```diff
@@ -0,0 +1,30 @@
+// interface-extension.slang
+
+// Test that an `extension` applied to an interface type works as users expect
```

Comment:
These comments are little confusing.
I assume that they are copied from another file?
If yes, you may need to modify these.

Reply:
Good catch, thanks!